.github/CONTRIBUTOR_AGREEMENT.md

```diff
@@ -87,7 +87,7 @@ U.S. Federal law. Any choice of law rules will not apply.
 7. Please place an “x” on one of the applicable statement below. Please do NOT
 mark both statements:
 
-    * [ ] I am signing on behalf of myself as an individual and no other person
+    * [X] I am signing on behalf of myself as an individual and no other person
```

Comment:
It looks like you accidentally filled in the contributor agreement template, instead of copying it over to `.github/contributors/demongolem.md`. Could you update that?